apps/storybook.namekit.io/stories/Namekit/DisplayedPrice.stories.tsx

```diff
@@ -0,0 +1,694 @@
+import {
+  CurrencySymbol,
+  CurrencySymbolPosition,
+  DisplayedPrice,
+  PriceDisplaySize,
+} from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+  Currency,
+  CurrencySymbology,
+  getCurrencySymbology,
+} from "@namehash/ens-utils";
+import React from "react";
+
+const meta: Meta<typeof DisplayedPrice> = {
+  component: DisplayedPrice,
+  title: "Namekit/DisplayedPrice",
+  argTypes: {
+    price: {
+      control: "object",
```

Comment:
Let's please better configure our Storybook stories so that we never have controls of type "object".

![CleanShot 2024-08-25 at 18 48 00](https://github.com/user-attachments/assets/b24bc29c-92be-4f8b-aede-f27f48658be4)

These "object" controls aren't usable and provide a bad UX in storybook.

One of the goals of Storybook is to ensure its easy for people to use the controls to see the UI component in various states.

This means you need to separate the idea of the schema for the controls in Storybook from the schema of the props that are actually passed into the constructor of the UI component. Please see the docs for Storybook they have examples for how to do this.

We should see a control in storybook that has a nice UX for picking the currency and a different control that has a nice UX for picking the price value.

You might want to consider also making it so that the control that picks the price value is just used as input to the `numberAsPrice` function to get the `Price` object that we actually pass into `DisplayedPrice`. Otherwise people can get confused when they first look at the storybook without understanding all the technical background.

Please ask me if any questions.

Reply:
Not sure how to do it but will manage doing so :)

Reply:
@FrancoAguzzi Check out the docs for storybook. That should give some ideas.

Reply:
@lightwalker-eth the best way to provide users the ability to set such arguments (React.ReactNode ones or typed Objects) is to provide multiple options for them. I have created multiple symbol and price options so people can test alternative examples of our Stories. Please let me know if you would like to see different options in our documentation!